app/featureset/featureset.go

```diff
@@ -38,12 +38,15 @@ type Feature string
 const (
 	// QBFTConsensus introduces qbft consensus, see https://github.com/ObolNetwork/charon/issues/445.
 	QBFTConsensus Feature = "qbft_consensus"
+	// DKG introduces Frost DKG algorithm into charon.
+	DKG Feature = "dkg"
```

Comment:
if frost, then probably name `frost_dkg`. Also, where do you plan to put the feature flag to enable/disable the feature?

Reply:
how feature flags marked as alpha are enabled/disabled